docs/configuration.md

```diff
@@ -229,3 +230,10 @@ If creating servers in your OpenStack takes a long time, you can increase the ti
 ## Custom pod network CIDR
 
 If `192.168.0.0/16` is already in use within your network, you must select a different pod network CIDR. You have to replace the CIDR `192.168.0.0/16` with your own in the generated file.
+
+## configuration of KubeadmControlPlane
+`KubeadmControlPlane` is defined [here](https://github.com/kubernetes-sigs/cluster-api/blob/master/controlplane/kubeadm/api/v1alpha3/kubeadm_control_plane_types.go#L146) and you need to define it to create a kubeadm control plane.
+
+`cluster-api-openstack-provider` defined templates for you at [with-lb](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/templates/cluster-template.yaml) and [without-lb](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/templates/cluster-template-without-lb.yaml), you can go ahead and do additional configuration.
```

Comment:
What is "additional configuration"?

Reply:
such as cloud specific settings ..I will make it more clear